joomla/joomla-cms
# Fixed failure on updating database not supporting UTF8MB4

## Problem

Users running a MySQL database that does not support UTF8MB4 will get failures during an update to Joomla 3.5.0 because the queries to update the tables to UTF8MB4 are executed.
## Fix

The fix is to modify the SQL queries that are run on databases not supporting UTF8MB4.
## Testing
1. Setup a Joomla 3.4.5 site with a MySQL database version that does not support UTF8MB4. For example mysqlnd lower than 5.0.9 or libmysql lower than 5.5.3.
2. Download the Joomla 3.5.0 beta 1 package
3. Change the files in the package with the files in this pull request
4. Update the Joomla version to 3.5.0 beta 1
5. The update should run without problems
## Disclaimer

You may still see database problems on the database tab due to different field definitions. This is not handled in this fix but will be dealt with in another fix.
